freqtrade/persistence/models.py

```diff
@@ -119,6 +119,7 @@ class Order(_DECL_BASE):
     ft_order_side: str = Column(String(25), nullable=False)
     ft_pair: str = Column(String(25), nullable=False)
     ft_is_open = Column(Boolean, nullable=False, default=True, index=True)
+    is_fully_realized = Column(Boolean, nullable=True, default=False)
```

Comment:
This property is pointless and tracks an irrelevant fact, please remove.

You will need additional columns to track realized profit (for ease of access from RPC methods) - but **not** on the Order object.

Reply:
Added a unit test for that.

Reply:
Done